packages/css/src/components/label/label.scss

```diff
@@ -17,3 +17,9 @@
   @include hyphenation;
   @include text-rendering;
 }
+
+.ams-label__heading {
+  font-size: initial;
+  font-weight: initial;
```

Comment:
We're setting initial values on a native `h1` here, that's wrapped around the Label. That doesn't seem to make sense. 

Shouldn't we set `var(--ams-label-font-size` and `var(--ams-label-font-weight)` here?